config/rwahs/components/providence

```diff
@@ -30,7 +30,7 @@ function postDeploy {
     ln -s "${localSettingsRoot}/setup.php" "${targetRoot}/${targetParent}/current/setup.php"
     ln -s "${localSettingsRoot}/media/collectiveaccess" "${targetRoot}/${targetParent}/current/media/collectiveaccess"
 
-    "${COLLECTIVEACCESS_HOME}/support/bin/caUtils" update-database-schema
+    yes | "${COLLECTIVEACCESS_HOME}/support/bin/caUtils" update-database-schema
```

Comment:
Note that the commands I'm using in the hooks aren't being pre-checked, so if they don't exist on the system you'll get an error midway through a deployment.  This isn't the best, ideally it would have another function that returns a list of commands that it requires, and then those would be checked by the `deploy` task.  But I don't know if that's worth doing.